RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -254,7 +406,68 @@
 #                                   jetCoreRegionalStepClassifier1,jetCoreRegionalStepClassifier2,
                                    jetCoreRegionalStep)
 JetCoreRegionalStep = cms.Sequence(JetCoreRegionalStepTask)
-seedingDeepCore.toReplaceWith(JetCoreRegionalStep,JetCoreRegionalStep.copyAndExclude([jetCoreRegionalStepHitDoublets]))
+
+JetCoreRegionalStepBarrelTask = cms.Task(jetsForCoreTrackingBarrel,
+                                         firstStepGoodPrimaryVertices,
+                                         #jetCoreRegionalStepClusters,
+                                         jetCoreRegionalStepSeedLayers,
+                                         jetCoreRegionalStepBarrelTrackingRegions,
+                                         jetCoreRegionalStepBarrelHitDoublets,
+                                         jetCoreRegionalStepSeedsBarrel,
+                                         jetCoreRegionalStepBarrelTrackCandidates,
+                                         jetCoreRegionalStepBarrelTracks,
+                                         #                                   jetCoreRegionalStepClassifier1,jetCoreRegionalStepClassifier2,
+                                         jetCoreRegionalStepBarrel)
+#JetCoreRegionalBarrelStep = cms.Sequence(JetCoreRegionalStepBarrelTask)
+
+JetCoreRegionalStepEndcapTask = cms.Task(jetsForCoreTrackingEndcap,
+                                         firstStepGoodPrimaryVertices,
+                                         #jetCoreRegionalStepClusters,
+                                         jetCoreRegionalStepSeedLayers,
+                                         jetCoreRegionalStepEndcapTrackingRegions,
+                                         jetCoreRegionalStepEndcapHitDoublets,
+                                         jetCoreRegionalStepSeedsEndcap,
+                                         jetCoreRegionalStepEndcapTrackCandidates,
+                                         jetCoreRegionalStepEndcapTracks,
+                                         #                                   jetCoreRegionalStepClassifier1,jetCoreRegionalStepClassifier2,
```

Comment:
https://github.com/cms-sw/cmssw/pull/33531/files#r625185765
please check

Reply:
not sure I understand

Reply:
> not sure I understand

the comment with code change suggestion was to remove the commented out line: - can it be removed; - if not, can the huge indentation be adjusted?

It may be more practical to read pending comments from https://github.com/cms-sw/cmssw/pull/33531/files
(the context is not fully visible from the main PR display https://github.com/cms-sw/cmssw/pull/33531)